ts/components/messages/MessageList/helpers.tsx

```diff
@@ -32,43 +34,48 @@ export const ItemSeparator = () => <ItemSeparatorComponent noPadded={true} />;
 export const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
 type RenderItemProps = {
-  hasPaidBadge: (id: MessageCategory) => boolean;
+  message: UIMessage;
+  hasPaidBadge?: (id: MessageCategory) => boolean;
```

Comment:
Is this needed? Seems like not being used in the component

Reply:
removed https://github.com/pagopa/io-app/pull/4319/commits/09ea7f59f9b25f3bf2c195d08a2bb64933df1eec